framework/src/modules/dpos_v2/module.ts

```diff
@@ -196,7 +196,10 @@ export class DPoSModule extends BaseModule {
 
 		this.endpoint.init(this._moduleConfig);
 
-		this._reportDelegateMisbehaviorCommand.init({ tokenIDDPoS: this._moduleConfig.tokenIDDPoS });
+		this._reportMisbehaviorCommand.init({
+			tokenIDDPoS: this._moduleConfig.tokenIDDPoS,
+			config: this._moduleConfig,
```

Comment:
Instead of passing the whole `this._moduleConfig` object here, you can pass only the other relevant missing config attribute, which is `this._moduleConfig.factorSelfVotes`.

Look at the line below, how it is implemented for `_unlockCommand`.